### PR TITLE
initial setup of client side done

### PR DESCRIPTION
**Pull request by aghilodessey**
-->Initial setup of client side done

### CODE REVIEW

File: client/src/routes/AuthenticatedRoutes.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+const AuthenticatedRoutes = () => {
+  return (
+    <div>
+      <Routes>
+        <Route path="/alerts" element={<Alerts />} />
+        <Route path="/dashboards" element={<Dashboards />} />
+        <Route path="/discover" element={<Discover />} />
+        <Route path="/issues" element={<Issues />} />
+        <Route path="/performance" element={<Performance />} />
+        <Route path="/projects" element={<Projects />} />
```
Comment (nihiannd) [inline client/src/routes/AuthenticatedRoutes.tsx:10]: Keep this `<Route path="/projects" element={<Projects />} />` and remove other routes

File: client/src/pages/alerts/Alerts.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/alerts/Alerts.tsx]: Remove this complete folder.
cmnt 101

File: client/src/pages/dashboards/Dashboards.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/dashboards/Dashboards.tsx]: cmnt 101

File: client/src/pages/discover/Discover.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/discover/Discover.tsx]: cmnt 101

File: client/src/pages/issues/Issues.tsx [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/issues/Issues.tsx]: cmnt 101

File: client/src/pages/performance/Performance.tsx [hunk 6]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/performance/Performance.tsx]: cmnt 101

File: client/src/pages/releases/Releases.tsx [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/releases/Releases.tsx]: cmnt 101

File: client/src/pages/replays/Replays.tsx [hunk 8]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/replays/Replays.tsx]: cmnt 101

File: client/src/pages/settings/Settings.tsx [hunk 9]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/settings/Settings.tsx]: cmnt 101

File: client/src/pages/stats/Stats.tsx [hunk 10]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/stats/Stats.tsx]: cmnt 101

File: client/src/pages/userfeedback/UserFeedBack.tsx [hunk 11]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
```
Comment (nihiannd) [inline client/src/pages/userfeedback/UserFeedBack.tsx]: cmnt 101

**Review by nihiannd — CHANGES_REQUESTED** [review]

File: client/src/routes/AuthenticatedRoutes.tsx (hunk 1 above)
Comment (aghilodessey) [reply client/src/routes/AuthenticatedRoutes.tsx:10]: Done

File: client/src/pages/alerts/Alerts.tsx (hunk 2 above)
Comment (aghilodessey) [reply client/src/pages/alerts/Alerts.tsx]: Done

File: client/src/pages/dashboards/Dashboards.tsx (hunk 3 above)
Comment (aghilodessey) [reply client/src/pages/dashboards/Dashboards.tsx]: Done

File: client/src/pages/discover/Discover.tsx (hunk 4 above)
Comment (aghilodessey) [reply client/src/pages/discover/Discover.tsx]: Done

File: client/src/pages/issues/Issues.tsx (hunk 5 above)
Comment (aghilodessey) [reply client/src/pages/issues/Issues.tsx]: Done

File: client/src/pages/performance/Performance.tsx (hunk 6 above)
Comment (aghilodessey) [reply client/src/pages/performance/Performance.tsx]: Done

File: client/src/pages/releases/Releases.tsx (hunk 7 above)
Comment (aghilodessey) [reply client/src/pages/releases/Releases.tsx]: Done

File: client/src/pages/replays/Replays.tsx (hunk 8 above)
Comment (aghilodessey) [reply client/src/pages/replays/Replays.tsx]: Done

File: client/src/pages/settings/Settings.tsx (hunk 9 above)
Comment (aghilodessey) [reply client/src/pages/settings/Settings.tsx]: Done

File: client/src/pages/stats/Stats.tsx (hunk 10 above)
Comment (aghilodessey) [reply client/src/pages/stats/Stats.tsx]: Done

File: client/src/pages/userfeedback/UserFeedBack.tsx (hunk 11 above)
Comment (aghilodessey) [reply client/src/pages/userfeedback/UserFeedBack.tsx]: Done

**Pull request merged**